### PR TITLE
Feature: Create, Display and Edit Rent #7

### LINKED ISSUE #7: [Gerät] Create 'Gerät ausleihen' view

### PR DESCRIPTION
**Pull request by PhilipW97**
closes #7 

The scope of this issue goes beyond just creating the view. It  enables users to create, display and edit a rent. 

I recommend to review commit by commit.

### CODE REVIEW

**Comment by PhilipW97** [comment]
Added input field to select hours
Bildschirmfoto 2021-06-10 um 16 24 25

**Comment by PhilipW97** [comment]
@pascalne2505 In the new rent view the user can only select Geräte that aren't rented currently.

File: src/main/java/com/surfSegelcenter/model/Price.java [hunk 1]
```diff
@@ -33,11 +35,14 @@
...
     )
     private Pricelist pricelist;
+    @OneToMany (mappedBy = "price")
+    private List<Rent> rents;
 
     public Price(double rentPrice, int rentDuration, Pricelist pricelist) {
         this.rentPrice = rentPrice;
         this.rentDuration = rentDuration;
         this.pricelist = pricelist;
+        this.rents = new ArrayList<>();
```
Comment (pascalne2505) [inline src/main/java/com/surfSegelcenter/model/Price.java:45]: is it necessary to store the rents in the price? whats the benefit?

File: src/main/java/com/surfSegelcenter/model/Rent.java [hunk 2]
```diff
@@ -184,7 +164,7 @@ public RentStatus getStatus() {
     public void setStatus(RentStatus rentStatus) {
         this.rentStatus = rentStatus;
         if(rentStatus.equals(RentStatus.BEENDET)) {
-            this.sportsequipment.updateProfitability(this.price);
+            // this.sportsequipment.updateProfitability(this.price);
             if(this.repCosts != 0){
```
Comment (pascalne2505) [inline src/main/java/com/surfSegelcenter/model/Rent.java:168]: what happens with the profitability if repCosts == 0?

Comment (PhilipW97) [reply src/main/java/com/surfSegelcenter/model/Rent.java:168]: Will improve this function in my repair PR,

**Review by pascalne2505 — APPROVED** [review]

**Pull request merged**